package.json

```diff
@@ -234,7 +256,9 @@
     "lint": "eslint src --ext ts",
     "watch": "tsc -watch -p ./",
     "pretest": "npm run test-compile && npm run lint",
-    "test": "node ./out/test/runTest.js"
+    "test": "node ./out/test/runTest.js",
+    "package": "vsce package",
+    "publish": "vsce publish"
```

Comment:
Could you add a new script below "publish" to run `prettier` on the `src` directory and run it? To keep the formatting consistent.
```
"prettier": "prettier --write src"
```

Reply:
Sure